docs/contributing/code-review.md

```diff
@@ -45,37 +45,42 @@ Here are some guidelines for merging and reviewing in this case:
   
 ## Self-merging as a community partner
 
-In some cases, as part of our [shared responsibility model](https://docs.2i2c.org/en/latest/about/service/shared-responsibility.html), we grant
-merge rights to partner engineers. Ideally, they would be only allowed to
-merge PRs related to their own cluster - we currently do not have an automatic
-way of enforcing this. 
+As part of our [shared responsibility model](https://docs.2i2c.org/en/latest/about/service/shared-responsibility.html), we may grant merge rights to partner engineers.
+This allows others to merge changes that impact their community's infrastructure without requiring intervention from a 2i2c engineer.
 
-```{note}
-While our current two partners who have such access (Cloudbank & Callysto)
-are grandparented in, we must have such technical enforcement mechanism
-in place before adding more such partners with merge rights
-```
+Merge rights are only given for partners with which we have built a relationship of trust, on a case-by-case basis.
+2i2c ultimately retains responsibility for the configuration of this infrastructure and its configuration.
+
+[See the `2i2c-org/collaborators` team](https://github.com/orgs/2i2c-org/teams/collaborators) for a list of sub-teams and individuals with write access to this repository.
 
-The goal here is that 2i2c is ultimately responsible for the configuration, so
-we do not want it to get too funky without 2i2c engineers knowing
-about it. As a community partner, if you think what you are doing
-*might* be funky, err on the side of asking for review!
+### Guidelines for community partners
+
+Our goal is to provide trusted communities the ability to more quickly make changes to their infrastructure in order to lead to a better, more collaborative service for their community.
+However, merge rights it a big responsibility, so please be careful in your actions.
```

Comment:
`is` a big responsibility 🤦 - but for some reason github isn't letting me suggest an edit 😕

Reply:
@choldgraf fixeed